gensim/models/word2vec.py

```diff
@@ -964,8 +953,8 @@ def train(
         Parameters
         ----------
         corpus_iterable : iterable of list of str
-            The `sentences` iterable can be simply a list of lists of tokens, but for larger corpora,
-            consider an iterable that streams the sentences directly from disk/network.
+            The `corpus_iterable` can be simply a list of lists of tokens, but for larger corpora,
```

Comment:
In general, Sphinx expects literals to be in double backticks.

```suggestion
            The ``corpus_iterable`` can be simply a list of lists of tokens, but for larger corpora,
```

There is no behavior for single backticks. We can customize that, but haven't, as far as I understand.

Reply:
OK, but this was just fixing the previous parameter name, leaving the prior-for-who-knows-how-long single-backtick formatting in place. Will commit your double-backticking.